src/App.js

```diff
@@ -96,7 +97,9 @@ class Root extends React.Component {
     hathorLib.WebSocketHandler.removeListener('is_online', this.isOnlineUpdate);
     hathorLib.WebSocketHandler.removeListener('reload_data', this.reloadData);
 
-    IPC_RENDERER.removeAllListeners("ledger:closed");
+    if (IPC_RENDERER) {
```

Comment:
Is it a bug fix? Maybe we should do it in another PR.

Reply:
It's not exactly a bug fix. This raises an error only when running in the browser (so only for dev). I would prefer not to have a PR only for this but if you think it's important, I can do it.

Reply:
I got it. No, I think it is good.